src/peerRatedPerformance.js

```diff
@@ -0,0 +1,239 @@
+import ForgeUI, { render, CustomField, CustomFieldEdit, Select, Option, useProductContext, useState, Text, StatusLozenge } from '@forge/ui';
+import api, {route} from '@forge/api';
+import { getCustomFieldID } from './index';
+
+const PeerAssessedPerformanceView = () => {
+    
+    const {platformContext: {issueKey}} = useProductContext();
+    const [assigneeId] = useState(getIssueAssigneeID(issueKey));
+    const {accountId} = useProductContext();
+    
+    // If current user logged in == assignee ID, then display “Cannot peer rate yourself”
+    if (accountId == assigneeId) {
+        return (
+            <CustomField>
+                <Text>Cannot peer rate yourself!</Text>
+            </CustomField>
+        )
+    };
+
+    // If ID of current user logged in does not exist in return value, then display “Peer assess performance”
+    const [fieldData] = useState(getPeerAssessedPerformanceData(issueKey));
+    var accounts;
+    var ratings;
+
+    if (fieldData === null || typeof fieldData === 'undefined') {
+        accounts = [];
+        ratings = [];
+    }
+    else {
+        accounts = fieldData.accountId;
+        ratings = fieldData.peerAssessedPerformanceRating;
+    };
+
+    var exists = false;
+    var counter = 0;
+    for (let i = 0; i < accounts.length; i++) {
+        if (accounts[i] == accountId) {
+            exists = true;
+            counter = i;
```

Comment:
Not sure if it would help much, but we can just `break;` here so we don't waste time going through other accounts?

Reply:
Yeah good catch! Added in a break here and somewhere else where I had a similar loop.